go/cli/internal/cli/chat.go

```diff
@@ -49,11 +50,31 @@ func ChatCmd(c *ishell.Context) {
 	if len(flagSet.Args()) > 0 {
 		teamName := flagSet.Args()[0]
 		var err error
-		team, err = client.GetTeam(cfg.UserID, teamName)
+		team, err = client.GetTeam(teamName, cfg.UserID)
 		if err != nil {
 			c.Println(err)
 			return
 		}
+		if team == nil {
+			c.Printf("Error: Could not find team '%s'\n", teamName)
+			teams, listErr := client.ListTeams(cfg.UserID)
+			if listErr != nil {
+				c.Printf("Failed to list teams: %v\n", listErr)
+				return
+			}
+
+			if len(teams) == 0 {
+				c.Println("No teams found, please create one via the web UI or CRD before chatting.")
+				return
+			}
+
+			c.Println("Available teams:")
+			if err := printTeams(teams); err != nil {
+				c.Printf("Failed to print teams: %v\n", err)
+				return
+			}
+			return
+		}
```

Comment:
Rather than printing teams, should this error move the user into the team selection logic? The same flow as the else below.